### DISCUSSION TITLE
The GitHub link on my website is experiencing functionality issues.

### DISCUSSION THREAD

**Discussion by alexcarey65** [community/community#81211] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've encountered a persistent issue with the GitHub link embedded within my website. Despite verifying the link's accuracy multiple times, it fails to redirect correctly, consistently resulting in a 404 error. This issue persists even after confirming the correctness of the URL.

Steps taken to troubleshoot:

Verified the correctness of the GitHub link in the website's source code.
Ensured that the linked repository exists and is publicly accessible.
Tried using different browsers and devices to access the link, but the issue persists across all.
Expected behavior:
When users click on the GitHub link on my website, they should be redirected to the corresponding GitHub repository without encountering any errors.

Actual behavior:
Clicking the GitHub link leads to a 404 error page, indicating that the content is not found.

I'm seeking guidance or suggestions on resolving this persistent issue. Has anyone else faced a similar problem with GitHub links on their website? Any insights, troubleshooting steps[,](https://capcutproapk.org/) or recommendations to rectify this issue would be greatly appreciated.

**Comment by ananuness** [comment]
Hey! Yesterday I saw some people commenting on this same problem... until now I haven't had it. If you feel comfortable, could you send the link you are trying to use?
